pulsar-version.txt

```diff
@@ -1 +1 @@
-2.5.0
+2.6.1
```

Comment:
ZSTD and SNAPPY seem to be available from Pulsar v2.6.0.
https://github.com/apache/pulsar/pull/7014

Reply:
Would you like me to change this to ```2.6.0``` instead of ```2.6.1```?

Reply:
Yes. I think the C++ client used for testing should be the lowest version that this library works with.